svelte-web-frontend/src/lib/pages/SettingsPage.svelte

```diff
@@ -31,6 +31,16 @@ $: pauseDurationStep = Math.round(10 * lerp(
 
 const qijiaScoreMin = 0;
 const qijiaScoreMax = 5;
+
+// Reset all variables to their default values
+function resetSettings() {
```

Comment:
Can we move the logic of the `resetSettings` function to `settings.ts`? As much as possible, I like to keep the business logic separate from the view logic.

Reply:
also a small nit: can we rename `resetSettings` to `resetSettingsToDefault` just to be crystal clear in the naming?